### ISSUE TITLE
Unsupperted URL Type "mongodb-query-parser" patch error

### ISSUE THREAD

**Issue by ilkeraltin** [mongo-express/mongo-express#1162]
Hi,

I'm trying to use `mongo-express` but it gives an error with `npm install` due to `mongodb-query-parser` dependency

<img width="822" alt="Screenshot 2023-03-22 at 08 14 47" src="https://user-images.githubusercontent.com/315650/226829005-3b4557ff-d0f1-4d8c-9047-75d7366639a3.png">

It works with `yarn` because that asks additional question to select your `mongodb-query-parser` version to install, this is fine for local env. but it fails on the pipeline.

<img width="824" alt="Screenshot 2023-03-22 at 08 15 34" src="https://user-images.githubusercontent.com/315650/226829221-25e3e755-ffaf-4837-a036-fe70699d4678.png">

Thanks!

**Comment by rtritto** [comment]
This issue is fixed with #1136.
Same problem with _renderjson_, that should be fix with #1146 (not yet merged).
Currently no new versions are released on npm registry.

FYI @BlackthornYugen

**Comment by ilkeraltin** [comment]
@rtritto Thanks for the quick reply. Looking forward to the new release on npm.

**Comment by olawalejuwonm** [comment]
Please when will this be merged? i can't use the package

**Comment by shakaran** [comment]
Merged in master, you can test it

**Comment by olawalejuwonm** [comment]
How do I install in master branch via npm?

**Comment by rtritto** [comment]
> How do I install in master branch via npm?

`npm i mongo-express@github:mongo-express/mongo-express`

**Comment by rtritto** [comment]
Waiting npm version > 1.0.1 to close this issue

**Comment by NeverBehave** [comment]
the npm version is 1.0.2 but this issue still persist

**Comment by rtritto** [comment]
@NeverBehave you are right, the https://github.com/mongo-express/mongo-express/commit/d96fdee14bfc25674a208107b7193f1e876f0836 is in the _1.0.3-rc_:
https://github.com/mongo-express/mongo-express/commits/v1.0.3-rc/?after=32f5b12c3a369c8f03c96e48f49c8d465d7c23c4+69

@BlackthornYugen please can you release a new version on `npm` or tell me how can I do?

**Comment by BlackthornYugen** [comment]
@rtritto I can invite you as a maintainer to the npm project. I don't think it's well documented how to release to npm. It would be great to automate it but the last couple releases I did from my machine. I don't remember how to do it, I'm pretty sure I just did npm --help. :)

**Comment by rtritto** [comment]
@BlackthornYugen to automate build of releases (GitHub and npm), we should use the file `/.github/workflows/publish.yml` where _NPM_TOKEN_ and _GITHUB_TOKEN_ secrets are used.

My username on npm is the same of GitHub.
